layouts/partials/archive-table.html

```diff
@@ -32,7 +32,7 @@
     {{if .resources.data}}
     <a href="/archive/{{.contest.year}}/{{.contest.type}}/{{.resources.data}}">
       <span class="material-symbols-outlined icon" title="All Data and Solutions">
-        task
+        folder_zip
```

Comment:
I like the original icon better. It is much clearer on this resolution size. @robertdijk  what do you think?
![image](https://user-images.githubusercontent.com/5221158/189710871-527e0959-01c6-4b4b-b697-e564d47f1cf3.png)

Reply:
I don't see any difference in that screenshot from what I normally see :thinking:

The reason why I chose `folder_zip`, is because it more closely represents what you get by clicking the link, namely, a zipped folder with the solutions :slightly_smiling_face:

Reply:
I spoke robert, he didn't mind either way. I still like the original better, but since its packaged with some other nice changes ill accept it.